### DISCUSSION TITLE
did unitas ever work for ultrakill? And if so, which versions of both?

### DISCUSSION THREAD

**Discussion by 10-days-till-xmas** [Eddio0141/UniTAS#326] [Q&A] [answered]
I heard that it had been used before so I'm just double checking

**Comment by Eddio0141** [comment, marked as answer]
just so others can also see the answer, version 0.5.1 and lower doesn't work with ultrakill, next release would have the working version
until then grab the latest build from https://github.com/Eddio0141/UniTAS/actions which has worked with ultrakill for some time
